src/me/mrCookieSlime/Slimefun/Setup/SlimefunManager.java

```diff
@@ -119,9 +120,11 @@ private static boolean equalsLore(List<String> lore, List<String> lore2) {
 		String string1 = "";
 		String string2 = "";
 		for (String string: lore) {
+			if (string.equals(ChatColor.GRAY + "Soulbound")) continue;
```

Comment:
Instead of adding a new check, take advantage of the check below, so change it to "&e&e&7Soulbound"
BTW it looks like the check below broke, as those & should be translated to color codes of course.